packages/venia-ui/lib/targets/venia-ui-intercept.js

```diff
@@ -25,4 +26,10 @@ module.exports = veniaTargets => {
         componentName: 'PlainHtmlRenderer',
         importPath: './plainHtmlRenderer'
     });
+
+    const paymentMethodList = new PaymentMethodList(venia);
+    paymentMethodList.add({
+        paymentCode: 'braintree',
+        importPath: './creditCard'
```

Comment:
It would be clear if you can either mention in a comment here that the path reference is from the `paymentMetodCollection.js` file or just change this to '@magento/venia-ui/lib/components/CheckoutPage/PaymentInformation/creditCard`.